script/upgrades/PoolRestructuring/PoolRestructuring.sol

```diff
@@ -0,0 +1,395 @@
+          biPoolManagerProxy,
+          abi.encodeWithSelector(IBiPoolManager(0).destroyExchange.selector, exchangeId, i)
+        )
+      );
+
+      if (poolsCleanupCfg.shouldRecreateWithNewSpread(currentExchange)) {
+        creations++;
+
+        IBiPoolManager.PoolExchange memory newExchange = poolsCleanupCfg.getPoolCfgWithNewSpread(currentExchange);
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            biPoolManagerProxy,
+            abi.encodeWithSelector(IBiPoolManager(0).createExchange.selector, newExchange)
+          )
+        );
+      }
+
+      if (i == 0) break;
+    }
+
+    require(
+      deletions == poolsCleanupCfg.poolsToDelete().length,
+      "❌ Number of deleted pools txs does not match expected"
+    );
+    require(
+      creations == poolsCleanupCfg.spreadOverrides().length,
+      "❌ Number of created pools txs does not match expected"
+    );
+  }
+
+  function updateValueDeltaBreakersThreshold() internal {
+    ValueDeltaBreakerCfg.Override[] memory overrides = valueDeltaBreakerCfg.valueDeltaBreakerOverrides();
+
+    for (uint256 i = 0; i < overrides.length; i++) {
+      uint256 currentThreshold = ValueDeltaBreaker(valueDeltaBreaker).rateChangeThreshold(overrides[i].rateFeedId);
+      require(currentThreshold == overrides[i].currentThreshold, "❌ Current threshold mismatch");
+    }
+
+    address[] memory rateFeedIds = Arrays.addresses(overrides[0].rateFeedId, overrides[1].rateFeedId);
+    uint256[] memory newThresholds = Arrays.uints(overrides[0].targetThreshold, overrides[1].targetThreshold);
+
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        valueDeltaBreaker,
+        abi.encodeWithSelector(ValueDeltaBreaker(0).setRateChangeThresholds.selector, rateFeedIds, newThresholds)
+      )
+    );
+  }
+
+  function updateTradingLimits() internal {
+    TradingLimitsCfg.Override[] memory overrides = tradingLimitsCfg.tradingLimitsOverrides();
+
+    for (uint256 i = 0; i < overrides.length; i++) {
+      bytes32 exchangeId = referenceRateFeedIDToExchangeId[overrides[i].referenceRateFeedID];
+      require(exchangeId != bytes32(0), "❌ Exchange ID not found for trading limits override");
+
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          brokerProxy,
+          abi.encodeWithSelector(
+            Broker(0).configureTradingLimit.selector,
+            exchangeId,
+            overrides[i].asset0,
+            TradingLimits.Config({
+              timestep0: overrides[i].asset0Config.timeStep0,
+              timestep1: overrides[i].asset0Config.timeStep1,
+              limit0: overrides[i].asset0Config.limit0,
+              limit1: overrides[i].asset0Config.limit1,
+              limitGlobal: overrides[i].asset0Config.limitGlobal,
+              flags: Config.tradingLimitConfigToFlag(overrides[i].asset0Config)
+            })
+          )
+        )
+      );
+
+      transactions.push(
```

Comment:
I think on the cEUR/axlEURC we don't have an asset1 limit configured not sure if that's a bad thing by itself @rcroessmann maybe should weigh in here, but if we want to keep it that way I guess we're creating a noop transaction here by configuring an empty trading limit over an already empty trading limit.

Reply:
Ended up adding the same limit on asset1 here, so 100K L0 and 500K L1 on axlEURC